teos/src/main.rs

```diff
@@ -103,12 +103,28 @@ async fn main() {
         .init()
         .unwrap();
 
+    // log default data dir
+    log::info!("Using default data directory: {:?}", &path);
+
+    // Log config file path
+    log::info!("Using configuration file: {:?}", &conf_file_path);
+
     if is_default {
         log::info!("Loading default configuration")
     } else {
         log::info!("Loading configuration from file")
     }
 
+    // logging the non-default options
+    for (key, (default_value, given_value)) in conf.get_non_default_options() {
+        log::info!(
```

Comment:
Seeing this now it looks way to verbose. My bad.

I think it'll be best if we just return the key and the modified value from `get_non_default_options` and we don't log what the expected value was.